### ISSUE TITLE
Latest version is still broken

### ISSUE THREAD

**Issue by adius** [tldr-pages/tldr-node-client#16]
```
$ npm install -g tldr
/usr/local/bin/tldr -> /usr/local/lib/node_modules/tldr/bin/tldr
tldr@1.0.0 /usr/local/lib/node_modules/tldr
├── ms@0.6.2
├── deep-extend@0.2.11
├── commander@2.3.0
├── rimraf@2.2.8
├── async@0.9.2
├── wrench@1.5.8
├── marked@0.3.3
├── colors@1.0.3
├── cpr@0.3.3 (graceful-fs@3.0.8)
├── mkdirp@0.5.1 (minimist@0.0.8)
├── unhtml@0.1.0 (quotemeta@0.0.0, html5-entities@0.5.1)
├── lodash@2.4.2
├── unzip@0.1.11 (setimmediate@1.0.2, pullstream@0.4.1, match-stream@0.0.2, readable-stream@1.0.33, fstream@0.1.31, binary@0.3.0)
└── request@2.44.0 (caseless@0.6.0, forever-agent@0.5.2, aws-sign2@0.5.0, stringstream@0.0.4, oauth-sign@0.4.0, tunnel-agent@0.4.0, json-stringify-safe@5.0.1, qs@1.2.2, node-uuid@1.4.3, mime-types@1.0.2, bl@0.9.4, form-data@0.1.4, http-signature@0.10.1, tough-cookie@2.0.0, hawk@1.1.1)

$ tldr --update
Updating...
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: stream.push() after EOF
    at readableAddChunk (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_readable.js:168:15)
    at SliceStream.Readable.push (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_readable.js:149:10)
    at SliceStream.Transform.push (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_transform.js:145:32)
    at SliceStream.sliceFn (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/pullstream/pullstream.js:81:14)
    at SliceStream._transform (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/pullstream/node_modules/slice-stream/slicestream.js:28:10)
    at SliceStream.Transform._read (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at SliceStream.Transform._write (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_transform.js:172:12)
    at doWrite (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_writable.js:237:10)
    at clearBuffer (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_writable.js:316:5)
    at onwrite (/usr/local/lib/node_modules/tldr/node_modules/unzip/node_modules/readable-stream/lib/_stream_writable.js:274:7)

$ tldr --update
Updating...
Error: invalid signature: 0xa0000
    at /usr/local/lib/node_modules/tldr/node_modules/unzip/lib/parse.js:59:13
    at processImmediate [as _immediateCallback] (timers.js:358:17)
```

**Comment by rprieto** [comment]
My bad, forgot to publish a new version after the fix. `v1.0.1` should fix that.

**Comment by igorshubovych** [comment]
@adius 

Please update tldr-node-client to 1.1.0:

`npm update -g`

And then run pages update again

`tldr --update`
